packages/assets-controllers/src/TokenBalancesController.ts

```diff
@@ -26,79 +20,76 @@ import { StaticIntervalPollingController } from '@metamask/polling-controller';
 import type {
   PreferencesControllerGetStateAction,
   PreferencesControllerStateChangeEvent,
-  PreferencesState,
 } from '@metamask/preferences-controller';
-import { isStrictHexString, type Hex } from '@metamask/utils';
-import type BN from 'bn.js';
-import type { Patch } from 'immer';
-import { isEqual } from 'lodash';
-
-import type { MulticallResult } from './multicall';
-import { multicallOrFallback } from './multicall';
-import type { Token } from './TokenRatesController';
+import type { Hex } from '@metamask/utils';
+import { isStrictHexString } from '@metamask/utils';
+import { produce } from 'immer';
+import isEqual from 'lodash/isEqual';
+
+import type { AccountTrackerUpdateNativeBalanceAction } from './AccountTrackerController';
+import {
+  AccountsApiBalanceFetcher,
+  type BalanceFetcher,
+  type ProcessedBalance,
+} from './multi-chain-accounts-service/api-balance-fetcher';
+import { RpcBalanceFetcher } from './rpc-service/rpc-balance-fetcher';
 import type {
   TokensControllerGetStateAction,
   TokensControllerState,
   TokensControllerStateChangeEvent,
 } from './TokensController';
+// endregion
 
-const DEFAULT_INTERVAL = 180000;
+// ────────────────────────────────────────────────────────────────────────────
+// region: Type aliases & constants
```

Comment:
Do we need these? TBH I'm not against it if it helps segment and improves readability.